navigator_engine/common/graph_loader.py

```diff
@@ -201,3 +216,25 @@ def _map_excel_boolean(boolean):
     else:
         raise ValueError('Value {} read from Initial Graph Config is not valid; Only TRUE and FALSE are valid values.'
                          .format(boolean))
+
+
+def _parse_resources(resource_cell):
+    resources = []
+
+    if pd.isnull(resource_cell):
+        return resources
+
+    resource_rows = re.split('\n', resource_cell)
+    for row in resource_rows:
+        url_re = re.compile(r'(https?://[^\s]+)')
+        search_results = url_re.search(row)
+        url = search_results.group(0)
+        title = row[0:search_results.span()[0]].strip()

```

Comment:
Can we express this is a more simple manner?

```suggestion
        title = row.split('http')[0].strip()
        url = row.split(title)[1].strip() 
```

Reply:
I've used regexp since it leaves open the option to do more sophisticated URL pattern matching. I started with this regexp https://stackoverflow.com/questions/161738/what-is-the-best-regular-expression-to-check-if-a-string-is-a-valid-url but it would require some work to get it to work in Python. I didn't consider it a priority. The current solution will skip e.g. any FTP addresses though.

Reply:
I would just prioritise "simple" in this context.